developer-hub/src/polkadot_sdk/frame_runtime.rs

```diff
@@ -15,7 +15,7 @@
 //!
 //! ## Introduction
 //!
-//! As described in [`crate::reference_docs::wasm_meta_protocol`], at a high-level Substrate-based
+//! As described in [`crate::reference_docs::wasm_meta_protocol`], at a high-level substrate-based
```

Comment:
why the lowercase? I think we should stick to uppercase as it is a product name 

```suggestion
//! As described in [`crate::reference_docs::wasm_meta_protocol`], at a high-level Substrate-based
```

Reply:
oops, typo 🙈